Ports/.port_include.sh

```diff
@@ -824,7 +824,9 @@ do_dev() {
 
     local first_hash="$(git -C "$git_repo" rev-list --max-parents=0 HEAD)"
 
+    pushd "$workdir"
     launch_user_shell
+    popd
```

Comment:
Let's silence this (but keep the pushd output, or silence that too and explicitly print out "switching to yay directory"):
```suggestion
    popd >/dev/null 2>&1
```

Reply:
:facepalm: what this guy said^

Reply:
I decided to only silence the `popd` command for now. Having an entirely custom message would likely look nicer, but I (and others as well) would probably like to see an error message in case anything goes wrong with the ports that are doing weird stuff with their `workdir` (:yakplus:).